tests/php/i18n/i18nTextCollectorTest.php

```diff
@@ -300,9 +300,14 @@ public function testCollectFromCodeNamespace()
 <?php
 namespace SilverStripe\Framework\Core;
 
+use SilverStripe\ORM\DataObject;
+
 class MyClass extends Base implements SomeService {
     public function getNewLines(\$class) {
-        if (!is_subclass_of(\$class, DataObject::class) || !Object::has_extension(\$class, Versioned::class)) {
+        if (
+            !is_subclass_of(\$class, DataObject::class)
+            || !Object::has_extension(\$class, \SilverStripe\Versioned\Versioned::class)
+        ) {
```

Comment:
Is it important for the test to have one namespace used via the use statement and one without it for the test to fail? If both were done the same way, would the test still work as expected, @bimthebam?

Reply:
It is, because both of them will affect the generated namespace for the language file. Have a look at my issue #10227 please. The use statement (`T_NAME_QUALIFIED`) and the direct usage of `\SilverStripe\i18n\i18n::get_locale();` (`T_NAME_FULLY_QUALIFIED`) ensure that the generated namespace is corrupted.